config.go

```diff
@@ -33,6 +36,17 @@ type Process struct {
 	Cwd string `json:"cwd"`
 }
 
+// User specifies linux specific user and group information for the container's
+// main process.
+type User struct {
```

Comment:
+1 and while in there perhaps renaming "Linux" above too - when I first saw `Linux Linux ...` it made me laugh.